spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/LocalDeployerProperties.java

```diff
@@ -67,7 +67,8 @@
 	 * Array of regular expression patterns for environment variables that
 	 * should be passed to launched applications.
 	 */
-	private String[] envVarsToInherit = {"TMP", "LANG", "LANGUAGE", "LC_.*", "PATH"};
+	private String[] envVarsToInherit = {"TMP", "LANG", "LANGUAGE", "LC_.*", "PATH",
+			"SPRING_CLOUD_APPLICATION_GUID", "SPRING_APPLICATION_INDEX", "INSTANCE_INDEX"};
```

Comment:
this was necessary due to a change where we moved adding these env vars earlier in the process instead of setting them directly on the process builder (which didn't work for the docker scenario since the env vars wasn't passed in to the container). Also, the way the env vars are handled is subtractive, so unless we add these env vars to `envVarsToInherit` they get removed. Maybe we should rename the variable to be `envVarsToKeep`